frontend/src/lib/components/Playlist/Playlist.scss

```diff
@@ -51,7 +50,7 @@
 
 .SessionRecordingPlaylistHeightWrapper {
     // NOTE: Somewhat random way to offset the various headers and tabs above the playlist
-    height: calc(100vh - 15rem);
+    height: calc(100vh - 10rem);
```

Comment:
dropping into both of our heads that it would be so cool to be able to remove this and just have this as a min-height and we fill the viewport otherwise

(although that can come... doesn't have to be in this PR 😊)